src/components/ComponentSettings/components/ComponentSettingsNotAvailable/ComponentSettingsNotAvailable.tsx

```diff
@@ -0,0 +1,36 @@
+import { transparentize } from 'polished';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@keen.io/colors';
+import { BodyText } from '@keen.io/typography';
+import { PickerWidgets } from '@keen.io/widget-picker';
+
+type Props = {
+  widgetType?: PickerWidgets;
+};
+
+const ComponentSettingsNotAvailable = ({ widgetType }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <BodyText
+      variant="body1"
+      style={{
+        height: '100%',
+        display: 'flex',
```

Comment:
Using `pure` CSS in style property do not provide all auto-prefixes for cross browser compatibility. I suggest to move flex to separate styled component.